docs/modules/ROOT/pages/how-to/day2ops/configure-kibana.adoc

```diff
@@ -0,0 +1,28 @@
+= Configure Kibana Index Patterns
+
+This guide describes how to configure Kibana index patterns.
+This must be done once after installing the OpenShift EFK logging stack.
```

Comment:
As far as I know, index patterns are stored on the user level. In consequence, they have to be done by each and every user individually.